pkg/cmd/pr/checks/checks.go

```diff
@@ -176,6 +181,14 @@ func checksRun(opts *ChecksOptions) error {
 		return err
 	}
 
+	if len(checks) == 0 {
+		if opts.Exporter != nil {
+			return opts.Exporter.Write(opts.IO, []check{})
+		}
+		fmt.Fprintf(opts.IO.Out, "No checks reported on the '%s' branch\n", pr.HeadRefName)
+		return cmdutil.NoChecksError
+	}
+
```

Comment:
This exits `1` when no checks are found. That is not the behavior we want. If no checks are found, we still exit `0`, as the command ran successfully.